metasyn/schema/plugin_registry.toml

```diff
@@ -0,0 +1,5 @@
+[disclosure]
+
+url = "https://github.com/sodascience/metasyn-disclosure-control"
+privacy = ["disclosure"]
+distributions = []
```

Comment:
I don't want to have to reproduce the whole DistributionProvider though. The way it will be used is to tell the users if the distribution cannot be found, that there is another plugin that might have it. So I don't think for that we really need to have all distributions, just the distributions that are not in core.